app/views/cloud_volume/new.html.haml

```diff
@@ -1,16 +1 @@
-%form#form_div{:name           => "angularForm",
-               'ng-controller' => "cloudVolumeFormController as vm",
-               "ng-show"       => "vm.afterGet",
-               "ng-cloak"      => "",
-               "miq-form"      => true,
-               "model"         => "vm.cloudVolumeModel",
-               "model-copy"    => "vm.modelCopy"}
-  = render :partial => "layouts/flash_msg"
-  .form-horizontal
-    = render :partial => "common_new_edit"
-
-:javascript
-  ManageIQ.angular.app.value('cloudVolumeFormId', 'new');
-  ManageIQ.angular.app.value('storageManagerId', '#{@storage_manager.try(:id).to_s || "undefined"}');
```

Comment:
This was a feature to preset the selected ems when creating a new cloud volume from a nested list, is that intentionally gone?

Reply:
Is there a way to reproduce this case?

Reply:
Sure, go to a nested list under a provider, and create a volume from the toolbar there.

Reply:
Gotcha, @DavidResende0 you will need to pass an additional prop on the form component in the `new.html.haml` and consume it on the other side.

Reply:
Also you no longer need this line and the `:javascript` above is also obsolete...